docs/Tutorial_MPDist.ipynb

```diff
@@ -0,0 +1,309 @@
+{
```

Comment:
<p>I don't think we need to say this here as long as we understand that what we are doing is "correct" and the outcome is as expected</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/433/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>Hmm, I can remove it but it might confuse some readers if they see that the two do not match</p>

Reply:
<p>Okay, then I would omit "a mistake in the paper". I think that we can explain the difference without sounding like we are assigning fault. Does that make sense?</p>

Reply:
Yes, makes sense!